orb-slot-ctrl/Cargo.toml

```diff
@@ -18,8 +18,15 @@ eyre.workspace = true
 libc.workspace = true
 orb-build-info.path = "../build-info"
 thiserror.workspace = true
+tempfile = "3.12.0"
 
 [build-dependencies]
 orb-build-info = { path = "../build-info", features = ["build-script"] }
 
 [package.metadata.deb]
+
+[features]
+test = []
+
+[package.metadata.orb]
+unsupported_targets = ["aarch64-apple-darwin", "x86_64-apple-darwin"]
```

Comment:
It's not due to tests though, it's due to the implementation which uses `libc` (you do have an upcoming change in the other PR). So the bin for it wouldn't even run in macOS.

I'll leave this as is for now as you can remove it once you've merged the removal of `libc`. I am not a fan of annotating every single test we have because the macOS pipeline can't run it tbh :/